presto-product-tests/src/main/java/com/facebook/presto/tests/jdbc/PreparedStatements.java

```diff
@@ -215,16 +215,16 @@ public void preparedInsertApi()
                             127,
                             32767,
                             2147483647,
-                            new BigInteger("9223372036854775807"),
+                            new Long("9223372036854775807"),
                             Float.valueOf("123.345"),
                             234.567,
-                            BigDecimal.valueOf(345.678),
-                            BigDecimal.valueOf(345.678),
+                            BigDecimal.valueOf(345),
+                            new BigDecimal("345.67800"),
```

Comment:
That would probably work too, but this shows the value exactly as Tempto displays it if the row doesn't match.